src/views/v2/components/backlink.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * View: backlink
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/components/backlink.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://evnt.is/1aiy
+ *
+ * @version TBD
+ *
+ * @var array $backlink An array of data for breadcrumbs.
+ */
+
+if ( empty( $backlink['url'] ) || empty( $backlink['label'] ) ) {
+	return;
+}
+
+?>
+<nav class="tribe-events-back" aria-label="<?php esc_attr_e( 'Back link', 'the-events-calendar' ); ?>">
+	<a href="<?php echo esc_url( $backlink['url'] ); ?>" class="tribe-events-c-back-link tribe-common-anchor">
+		&larr; <?php echo esc_html( $backlink['label'] ); ?>
```

Comment:
Single event view uses `&laquo;` (double caret left) - we should match for consistency.

https://github.com/the-events-calendar/the-events-calendar/blob/main/src/views/single-event/back-link.php#L23 


```suggestion
		&laquo; <?php echo esc_html( $backlink['label'] ); ?>
```